source/docs/networking/networking-introduction/fms-network-architecture.rst

```diff
@@ -0,0 +1,111 @@
+# Understanding the FMS Network Architecture
+
+This article provides an overview of how the Field Management System (FMS) network is structured at FRC competitions. Understanding this architecture helps teams troubleshoot connectivity issues and design robust networking solutions for their robots.
+
+.. note:: This article summarizes key information from the `FMS Whitepaper <https://fms-manual.readthedocs.io/en/latest/fms-whitepaper/fms-whitepaper.html>`__. For complete technical details, refer to the full whitepaper.
+
+## Network Topology Overview
+
+The FMS network is an Ethernet-based system that connects all robots, driver stations, and field equipment. The **Score Switch** connects the following components:
+
+- **Field Router**: Connects to the FMS server and manages alliance networks
+- **FMS Server**: Controls match flow and communicates with Driver Station software
+- **Field Access Point (AP)**: Provides 6 GHz wireless connectivity to robots
+- **Smart Router**: Manages external connectivity and network services
+
+Robot-to-field communication uses a dedicated wireless access point operating in the 6 GHz band with 802.11ax (Wi-Fi 6E) to minimize interference from spectator devices.
+
+## Team Network Isolation
+
+The FMS network uses Virtual LANs (VLANs) to completely isolate each team's traffic. Each team's network contains **only** their robot, driver station, and the FMS server.
+
+.. important:: Teams cannot communicate with other teams' robots or driver stations, even during the same match. This isolation protects against accidental interference and ensures fair competition.
+
+## IP Addressing on the Field
+
+When connected to FMS, teams use the standard ``10.TE.AM.xx`` addressing scheme:
+
+**Field Network (DHCP Range):**
+- Driver Station: ``10.TE.AM.20`` - ``10.TE.AM.199`` (assigned by FMS)
+- Gateway: ``10.TE.AM.4``
+- Subnet Mask: ``255.255.255.0``
+
+**Robot Network (via Team Radio):**
+- Robot Radio: ``10.TE.AM.1`` (static)
+- roboRIO: ``10.TE.AM.2`` (DHCP from robot radio)
+- Other robot devices: ``10.TE.AM.200`` - ``10.TE.AM.219`` (DHCP from robot radio)
+
+.. note:: The Field Router does **not** provide DHCP services to robot devices. Your team's robot radio provides DHCP addressing for devices onboard the robot.
+
+## Wireless Specifications
+
+FMS wireless uses modern standards to ensure reliable connectivity:
+
+- **Frequency Band**: 6 GHz exclusively (avoids 2.4/5 GHz congestion from phones, tablets)
+- **Standard**: 802.11ax (Wi-Fi 6E)
+- **Encryption**: WPA3 with AES encryption
+- **Security Keys**: Unique per team, per event (teams configure radios using event kiosks)
+
+.. warning:: Field staff devices use separate 2.4 GHz and 5 GHz networks. Robot radios must be configured for 6 GHz operation at competition.
```

Comment:
Teams can't configure their own radios and customize the band. This seems like it can be deleted.